app/jobs/iiif_print/jobs/create_relationships_job.rb

```diff
@@ -71,12 +72,21 @@ def completed_child_data
       end
 
       def find_children_by_title_for(title, model)
-        # We should only find one, but there is no guarantee of that and `:where` returns an array.
-        model.constantize.where(title: title)
+        work_type = model.constantize
+        if work_type.respond_to?(:where)
```

Comment:
We could extract this method to the adapter logic:  

- lib/iiif_print/persistence_layer/active_fedora_adapter.rb
- lib/iiif_print/persistence_layer/valkyrie_adapter.rb